packages/panels/src/components/drawer/Drawer.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+export type SlideFrom = "left" | "right";
+
+export interface DrawerProps {
+  /** Ref to use for drawer. */
+  innerRef?: React.Ref<HTMLDivElement>;
+
+  portalTarget?: HTMLElement | null;
+  /* Whether the drawer is open or not. */
+  isOpen: boolean;
+  /**
+   * Which direction the drawer will appear from.
+   * @default left
+   * @param {String('left'|'right')}
+   */
+  slideFrom?: SlideFrom;
+  /**
+   * The width of the drawer.
+   * @default 400px
+   */
+  width?: string;
+  /**
+   * The zIndex for the drawer.
+   * @default 1000
+   */
+  zIndex?: number;
+}
+
+type DrawerWrapperProps = Pick<
+  DrawerProps,
+  "isOpen" | "slideFrom" | "width" | "zIndex"
+>;
+
+const DrawerWrapper = styled("div")<DrawerWrapperProps>`
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  transform: translateX(
+    ${({ isOpen, slideFrom, width }) =>
+      isOpen ? 0 : slideFrom === "right" ? `${width}` : `-${width}`}
+  );
+  right: ${props => (props.slideFrom === "right" ? 0 : "auto")};
+  left: ${props => (props.slideFrom === "left" ? 0 : "auto")};
+  height: 100%;
+  width: ${({ width }) => width};
+  transition: 0.6s all;
```

Comment:
0.6s is too slow. I've always done 0.25 for normal animations.

Reply:
Talked to Ozcar and he thought 0.25 was better. I'm changing!